Modix.Services/Promotions/PromotionsService.cs

```diff
@@ -149,6 +149,11 @@ public class PromotionsService : IPromotionsService
             var rankRoles = await GetRankRolesAsync(AuthorizationService.CurrentGuildId.Value);
             var subject = await UserService.GetGuildUserAsync(AuthorizationService.CurrentGuildId.Value, subjectId);
 
+            if (subject.IsBot)
+            {
+                return;
```

Comment:
If we just `return`, there isn't any indicator to the user that the command failed due to the nominee being a bot account - it just looks like Modix didn't even receive the command. Might be good to bail out with a message, possibly something like
```suggestion
                throw new InvalidOperationException("Bots cannot be nominated for a promotion campaign.");
```